BetterMD/elements/table.py

```
@@ -1,113 +1,487 @@
+        return f"**{content}**".center(width)
 
-class TdRST(CustomRst):
-    def to_rst(self, inner: list[Symbol], symbol: Symbol, parent: Symbol) -> str:
-        if len(inner) > 1 or not isinstance(inner[0], (Text, H1, H2, H3, H4, H5, H6)):
-            raise TypeError("Table Data may only contain text as inner")
+class ThRST(CustomRst):
+    def to_rst(self, inner, symbol, parent):
+        return " ".join([e.to_rst() for e in inner])
 
-        return inner[0].to_rst()
+class TBodyRST(CustomRst):
```

Ruff (0.8.2)

Redefinition of unused `TBodyRST` from line 99

(F811)